script/hassfest/quality_scale.py

```diff
@@ -221,6 +221,7 @@ class Rule:
     "bond",
     "bosch_shc",
     "braviatv",
+    "briiv",
```

Comment:
Please add a quality scale file instead. New integrations require Bronze tier:

https://developers.home-assistant.io/docs/core/integration-quality-scale/

https://developers.home-assistant.io/docs/core/integration-quality-scale/#keeping-track-of-the-implemented-rules